src/main/java/org/broadinstitute/hellbender/tools/walkers/concordance/Concordance.java

```diff
@@ -0,0 +1,244 @@
+            if (variant.isNotFiltered()) {
+                if (variant.isSNP()) { snpFalsePositives++; } else { indelFalsePositives++; }
+                writeThisRecord(new VariantStatusRecord(variant, FALSE_POSITIVE, evalSampleName), variant);
+            }
+
+            // we don't care about true negatives - don't record them in the table
+            return;
+        }
+    }
+
+    // ugly code alert
+    // this method just moves the iterator forward but also ensures that it skips SV's
+    // assumes that the caller already checked the truth iterator has next
+    // TODO: extend the truth iterator such that the iterator.next() automatically skips SVs
+    private Optional<VariantContext> getNextNonSVTruthVariant(){
+        VariantContext nextVariant;
+        do {
+            // if the truth vcf ends with an SV, truthIterator.next() may return null
+            // so make sure that we didn't reach the end of truth
+            if (! truthIterator.hasNext()){
+                exhaustedTruthVariants = true;
+                return Optional.empty();
+            }
+            nextVariant = truthIterator.next();
+        } while (nextVariant.getStructuralVariantType() != null);
+
+        return Optional.of(nextVariant);
+    }
+
+    private void writeThisRecord(final VariantStatusRecord record, final VariantContext variant){
+        try {
+            variantStatusWriter.writeRecord(record);
+        } catch (Exception e) {
+            throw new UserException(String.format("Encountered an IO exception writing a record at chrom %d, pos ",
+                    variant.getContig(), variant.getStart()), e);
+        }
+    }
+
+    @Override
+    public Object onTraversalSuccess(){
+        if (! exhaustedTruthVariants){
+            // the remaining variants in truth are false negatives
+            if (currentTruthVariant.get().isSNP()) { snpFalseNegatives++; } else { indelFalseNegatives++; }
+            while (truthIterator.hasNext()){
+                currentTruthVariant = getNextNonSVTruthVariant();
+                if (! currentTruthVariant.isPresent()) { break; }
+                if (currentTruthVariant.get().isSNP()) { snpFalseNegatives++; } else { indelFalseNegatives++; }
+            }
+        }
+
+        try ( ConcordanceSummaryRecord.Writer concordanceSummaryWriter = ConcordanceSummaryRecord.getWriter(summary) ){
+            // TODO: write all the summary
+            concordanceSummaryWriter.writeRecord(new ConcordanceSummaryRecord(VariantContext.Type.SNP, snpTruePositives, snpFalsePositives, snpFalseNegatives));
+            concordanceSummaryWriter.writeRecord(new ConcordanceSummaryRecord(VariantContext.Type.INDEL, indelTruePositives, indelFalsePositives, indelFalseNegatives));
+
+            // We must close the variant status writer to flush the buffer.
+            // Summary writer will be closed implicitly
+            variantStatusWriter.close();
```

Comment:
Yea I agree. I wanted to put it in `finally` block but turns out `close()` throws an `IOException`. So I think I need to put it in a try-catch.